src/sql/base/browser/ui/panel/media/panel.css

```diff
@@ -98,7 +97,7 @@ panel {
 	padding-left: 5px;
 	padding-right: 5px;
 	min-width: 65px;
-	flex: 0 0 auto;
+	/* flex: 0 0 auto; */
```

Comment:
If you're removing something just delete it

Reply:
Sorry, it seems I did not push the final code.